src/diffusers/models/cross_attention.py

```diff
@@ -48,19 +48,25 @@ def __init__(
         cross_attention_dim: Optional[int] = None,
```

Comment:
This class should essentially stay exactly as is, but happy to leave for now and help you afterwards with the weight conversion. If you transform the weights correctly Conv2D and nn.Linear is identical.

The reason we use `nn.Linear` for attention is because: a) it's faster and b) attention is supposed to be linear (q,k,v are linear projections).

Happy to explain more in a call :-)